src/components/Main/Schedule/services.ts

```diff
@@ -69,21 +69,24 @@ export function everyRepeatingDayBetweenTwoDates(
   return everyRepeatingDay;
 }
 
-const zeroingDates = (num1: number, num2: number): string => {
-  num1++;
-  const month = num1.toString().length > 1 ? num1.toString() : `0${num1.toString()}`;
-  const day = num2.toString().length > 1 ? num2.toString() : `0${num2.toString()}`;
+const zeroPaddingDates = (monthIdx: number, dayIdx: number): string => {
+  let month = (monthIdx++).toString();
```

Comment:
Does this work? if i try `(1++).toString()` in dev tools, i get an error (that's why i put `+1`)
```suggestion
  let month = (monthIdx+1).toString();
```

Reply:
ahh. I like ++ which is why I tried that, but I realized it was wrong. Thanks!